plugin.go

```diff
@@ -256,6 +265,16 @@ func checkoutSha(commit string) *exec.Cmd {
 	), defaultEnvVars...)
 }
 
```

Comment:
Is there no way to fetch by smal commit id?

If fetching by branch is done it could happen tjat restarts of pipelines produces different results

Reply:
Thanks for looking into it, I also did so but this looks like the best solution atm ...